src/hera.cpp

```diff
@@ -265,7 +266,8 @@ evmc_result hera_execute(
   memset(&ret, 0, sizeof(evmc_result));
 
   try {
-    heraAssert(rev == EVMC_BYZANTIUM, "Only Byzantium supported.");
+    (void)rev;
+//    heraAssert(rev == EVMC_BYZANTIUM, "Only Byzantium supported.");
```

Comment:
Why this is removed?

Reply:
For testing, because `geth`'s `evm` had no option to change it.